requirements.txt

```diff
@@ -11,9 +11,9 @@ applicationinsights
 prompt_toolkit
 rope
 tox
-pyyaml>=4.1,<=4.2b4
+pyyaml==5.4
 jsonpath_rw
-docker-compose>=1.21.0,<=1.24.0
+docker-compose==1.28.5
```

Comment:
>== [](start = 14, length = 2)

why fixing on this particular version and not this one or higher?

Reply:
same as above. didn't want to extrapolate beyond the component governance alert recommendations